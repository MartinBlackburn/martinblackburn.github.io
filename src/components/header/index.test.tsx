// libraries
import { render } from "@testing-library/react";

// component under test
import Header from "./";

// mocks
jest.mock("../typewriter", () => () => <div>typewriter</div>);

describe("Header component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<Header />);

        expect(asFragment()).toMatchSnapshot();
    });
});
