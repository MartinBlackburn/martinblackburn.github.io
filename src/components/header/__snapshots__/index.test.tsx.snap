// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<DocumentFragment>
  <header
    class="header"
    style="background-image: url(/images/background.jpg);"
  >
    <img
      alt="profile"
      src="/images/profile.jpg"
    />
    <div
      class="header__intro"
    >
      // Hello world
    </div>
    <h1
      class="header__name"
    >
      I'm Martin Blackburn
    </h1>
    <div>
      typewriter
    </div>
  </header>
</DocumentFragment>
`;
